cmd/nerdctl/network_rm.go

```diff
@@ -82,6 +82,11 @@ func networkRmAction(cmd *cobra.Command, args []string) error {
 			if err := os.RemoveAll(l.File); err != nil {
 				return err
 			}
+			// Remove the bridge network interface on the host.
+			if l.Plugins[0].Network.Type == "bridge" {
+				netIf := fmt.Sprintf("nerdctl%d", *l.NerdctlID)
```

Comment:
This should be fetched from here https://github.com/containerd/nerdctl/blob/ed3d553a9bd6818b0917f2e040e43df5973ff8c2/pkg/netutil/netutil_unix.go#L40

Reply:
It seems that there is no easy way to get the name of the bridge from plugins.

`l.Plugins[0].Network.Name` always is emtry.